.travis.yml

```
@@ -30,6 +37,30 @@ jobs:
 #  - stage: test
 #    dist: bionic
 #    python: 3.8
+  - stage: experimental
+    os: windows
+    name: "Python 3.6.7 on Windows"
+    language: sh
+    python: "3.6.7"
+    before_install:
+      - choco install python --version=3.6.7
+      - export PATH="/c/Python36:/c/Python36/Scripts:$PATH"
+      - python --version
+#      - choco install postgresql10 --params '/Password:' --paramsglobal
+    install:
+      - pip install numpy
+      - pip install scipy
+      - pip install --upgrade pytest
+      - pip install codecov
+      - if [ -z "$PANDAS" ]; then pip install pandas; else pip install pandas==$PANDAS; fi
```

Small readability tweak

```suggestion
      - pip install numpy scipy codecov
      - pip install --upgrade pytest
      - if [ -z "$PANDAS" ]; then pip install pandas; else pip install pandas==$PANDAS; fi
```